src/KeyVault/KeyVault/help/Add-AzKeyVaultKey.md

```diff
@@ -380,6 +383,44 @@ Tags           :
 
 Create a secure key in managed hsm named testmhsm. Its name is test-key and type is RSA. 
 
+### Example 10: Add a key for a Confidential VM to a key vault. 
+```powershell
+$keyVaultName = <Key Vault Name>
+$location = <Azure Region>
+$resourceGroupName = <Resource Group Name>
```

Comment:
There are some syntex check in example. Please make sure the example have the right syntex os PowerShell.

Reply:
looks like I was missing a ", should be fixed.